### PR TITLE
OSDOCS-1613 AWS permissions update

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.redhat.com/browse/OSDOCS-1613

### CODE REVIEW

**Comment by codyhoag** [comment]
@patrickdillon PTAL

**Comment by codyhoag** [comment]
@yunjiang29 can you confirm from a QE perspective?

**Comment by yunjiang29** [comment]
LGTM.

File: modules/installation-aws-permissions.adoc [hunk 1]
```diff
@@ -227,7 +226,7 @@ If you use an existing VPC, your account does not require these permissions for
 
 [NOTE]
 =====
-If you use an existing VPC, your account does not require these permissions to delete network resources.
+If you use an existing VPC, your account does not require these permissions to delete network resources. Instead, your account only requires the `tag:UntagResources` permission to delete network resources if you use an existing VPC.
```
Comment (jeana-redhat) [inline modules/installation-aws-permissions.adoc:229]: Seems you could omit the "if you use an existing VPC" and the end since the note starts with that.

**Review by jeana-redhat — APPROVED** [review]
Preview build won't load, but given the change I don't think I need it. LGTM

**Pull request merged**